external-crates/move/crates/move-cli/src/sandbox/commands/publish.rs

```diff
@@ -168,7 +165,7 @@ pub fn publish(
         let mut serialized_modules = vec![];
         for unit in modules_to_publish {
             let id = unit.unit.module.self_id();
-            let module_bytes = unit.unit.serialize(bytecode_version);
+            let module_bytes = unit.unit.serialize();
```

Comment:
Why not with_version? Is this test only code?

Reply:
This is a convenience method on `CompiledUnit` that uses `serialize_with_version` under the hood, but it's just so you don't need to do the whole `let mut bytes = vec![]; unit.unit.module.serialize_with_version(unit.unit.module.version, &mut bytes); bytes`. So I kinda liked keeping it. But I can remove if you want it gone completely.

Reply:
Ah this is on the compiler's unit, not the CompiledModule?

Reply:
yep! This is on `CompiledUnit` :)